.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
 install:
   - docker build -t savaslabs/squabble:latest .
```

Comment:
Just a thought, instead of `image:` in the web container in `docker-compose.yml`, could we have `build: source` instead? That would help automate some things and reduce error. Fine to keep as is though.